src/index.js

```diff
@@ -0,0 +1,141 @@
+  this.discription = discription;
+  this.completed = completed;
+  this.index = index;
+}
+
+btn.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const index = listArray.length + 1;
+  const completed = false;
+  if (listInput.value !== '' || null) {
+    const todoObject = new TodoListStore(listInput.value, completed, index);
+    listArray = [...listArray, todoObject];
+    localStorage.setItem('listArray', JSON.stringify(listArray));
+    UI.displayData();
+    UI.cleanInputs();
+  }
+});
+
+// Retrieve the listArray from Local Storage when the page is loaded..😎😊😉😎............
+
+// funtion for marking done lists😎😊😉😎............
+
+listHolder.addEventListener('change', (event) => {
+  if (event.target.type === 'checkbox' && event.target.checked) {
+    event.target.nextElementSibling.style.textDecoration = 'line-through';
+    event.target.completed = true;
+  } else if (event.target.type === 'checkbox' && !event.target.checked) {
+    event.target.nextElementSibling.style.textDecoration = 'none';
+  }
+});
+
+if (localStorage.getItem('listArray')) {
+  listArray = JSON.parse(localStorage.getItem('listArray'));
+  UI.displayData();
+}
+
+listHolder.addEventListener('click', (event) => {
+  if (event.target.classList.contains('fa-ellipsis-v')) {
+    event.target.classList.remove('fa-ellipsis-v');
+    event.target.classList.add('fa-trash');
+    event.target.parentElement.parentElement.style.backgroundColor = '#f39c12';
+    const targeted = event.target.parentNode.previousElementSibling.lastElementChild;
+    targeted.contentEditable = 'true';
+  } else if (event.target.classList.contains('fa-trash')) {
+    const index = event.target.parentElement.parentElement.getAttribute('data-index');
+    listArray.splice(index, 1);
+    localStorage.setItem('listArray', JSON.stringify(listArray));
+    event.target.parentElement.parentElement.remove();
+  }
+});
+listHolder.addEventListener('input', (e) => {
+  if (e.target.closest('#discription')) {
+    const inputText = e.target.closest('#discription');
+    const taskElm = inputText.parentNode.parentNode;
+    const index = Array.prototype.indexOf.call(listHolder.children, taskElm);
+    listArray[index].discription = e.target.textContent;
+
+    localStorage.setItem('listArray', JSON.stringify(listArray));
+  }
+});
+
+window.addEventListener('load', () => {
+  if (localStorage.getItem('listArray')) {
+    listArray = JSON.parse(localStorage.getItem('listArray'));
+    UI.displayData();
+  }
+});
+/*
+lists.forEach((list) =>{
+   list.addEventListener('click', () => {
+    console.log('clicked');
+   })
+})
+
+  Editing the list *********inputText.addEventListener(
+      'input',
+      (e) => {
+        const parent = document.querySelector('.list-container');
+        const child = e.target.closest('.task-container');
+        saveData(editText(loadData(), parent, child, inputText.textContent));
+      },
+      false,
+    );
```

Comment:
- [ ] Kindly remove commented code as this makes your codebase cleaner and more readable to other collaborators